### DISCUSSION TITLE
What does the circled bang mean?

### DISCUSSION THREAD

**Discussion by Pinaz993** [spacecowboy/Feeder#232] [Q&A] [answered]
![Screenshot_20240309-080256](https://github.com/spacecowboy/Feeder/assets/4016145/91f408c9-6787-4ba8-a661-9e02de8c0f46)

I can't find anywhere this is discussed. All I can think of is that something is wrong, but without more information, I can't figure out what.

**Comment by spacecowboy** [comment, marked as answer]
It's an error icon. It's shown because Feeder was unable to fetch the icon for the feed/site.

**Comment by Pinaz993** [comment]
Ah, understood. Strange that I've never seen an icon for the site before.
Is that a feature of Atom that RSS lacks?

**Reply by spacecowboy** [reply]
No. Nothing is shown if there is no icon specified. 

In this case the site is specifying an icon, either in the feed or as a favicon on the site, which is not a valid image.

**Comment by Pinaz993** [comment]
Makes sense, thanks for the prompt response.
